pennylane_lightning/lightning_qubit.py

```diff
@@ -34,33 +33,24 @@
 from ._version import __version__
 
 try:
-    if platform.system() == "Windows" and sys.version_info[:2] >= (3, 8):  # pragma: no cover
-        # Add the current directory to DLL path.
-        # See https://docs.python.org/3/whatsnew/3.8.html#bpo-36085-whatsnew
-        os.add_dll_directory(os.path.dirname(os.path.abspath(__file__)))
-        from lightning_qubit_ops import (
-            StateVectorC64,
-            AdjointJacobianC64,
-            VectorJacobianProductC64,
-            StateVectorC128,
-            AdjointJacobianC128,
-            VectorJacobianProductC128,
-        )
-    else:
-        from .lightning_qubit_ops import (
-            StateVectorC64,
-            AdjointJacobianC64,
-            VectorJacobianProductC64,
-            StateVectorC128,
-            AdjointJacobianC128,
-            VectorJacobianProductC128,
-        )
-    from ._serialize import _serialize_obs, _serialize_ops
+    from .lightning_qubit_ops import (
```

Comment:
No. I got confused as the Python error message just show the name of the original package even when it failed to load dependencies. So I thought it was a problem in importing `lightning_qubit_ops` but it was in fact dll files such as `libstdc++.dll`. I do not really understand how the previous `os.add_dll_directory` suppressed the error. It somehow made the statement raises `ModuleNotFoundError` (instead of `ImportError` that I get when a dll file is missing) and made `pennylane_lightning` to fall-back to the `default.qubit` (which I coulnd't catch as the `pl-test-device` passed).

Reply:
Windows is a challenge at the best of times :smile: